game.js

```diff
@@ -94,6 +95,29 @@ $(function () {
         }
     };
 
+    const potion = {
+        energy: 200,
+        thisImage: 0,
+        images: [
+            `${imgPathPotion}potion.png`,
+        ],
+        location: {
+            x: 600,
```

Comment:
The potion could be further away - 700 would be a better x location.

Reply:
The change was done, potion location now is at x:700.